TELCODOCS-1385 - Update docs for 4.14 RAN RDS (#64278)
openshift/openshift-docs: modules/telco-ran-hub-cluster-management.adoc

// Module included in the following assemblies:
//
// * telco_ref_design_specs/ran/telco-ran-du-overview.adoc

:_content-type: REFERENCE
[id="telco-ran-hub-cluster-management_{context}"]
= Hub cluster management resources

You manage the cluster configuration with {rh-rhacm-first} policies.
To meet CPU requirements for the hub cluster do the following:

> * Configure a maximum of 5 {rh-rhacm} policies with a compliant evaluation interval of not less than 10 minutes.

Reviewer: ```suggestion
* Configure a maximum of 5 {rh-rhacm} policies with a compliant evaluation interval of at least 10 minutes.
```
